.github/workflows/build-test-deploy.yml

```diff
@@ -81,33 +87,33 @@ jobs:
         run: mvn test --batch-mode -Dskip.npm
 
   build-docker-image:
-      needs: [build]
-      runs-on: ubuntu-latest
-      timeout-minutes: 30
-      env:
-        BUILD_NUMBER: ${{ needs.build.outputs.build_number }}
-        DOCKER_IMAGE_TAG: ${{ needs.build.outputs.docker_image_tag }}
-        GIT_COMMIT_HASH: ${{ github.sha }}
-      steps:
-        - uses: actions/checkout@v3
-
-        - name: Build Docker image
-          run: |
-            docker build --tag ${DOCKER_IMAGE_TAG} \
-            --build-arg versienummer=${BUILD_NUMBER} \
-            --build-arg buildId=${BUILD_NUMBER} \
-            --build-arg commit=${GIT_COMMIT_HASH} \
-            --file Containerfile \
-            .
-
-        - name: Save Docker Image
-          run: docker save --output docker-image.tar ${DOCKER_IMAGE_TAG}
-
-        - name: Cache Docker Image
-          uses: actions/cache/save@v3
-          with:
-            path: docker-image.tar
-            key: docker-image-${{ github.repository }}-${{ github.ref_name }}-${{ github.run_number }}
+    needs: [build]
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      BUILD_NUMBER: ${{ needs.build.outputs.build_number }}
+      DOCKER_IMAGE_TAG: ${{ needs.build.outputs.docker_image_tag }}
+      GIT_COMMIT_HASH: ${{ github.sha }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Build Docker image
+        run: |
+          docker build --tag ${DOCKER_IMAGE_TAG} \
+          --build-arg versienummer=${BUILD_NUMBER} \
+          --build-arg buildId=${BUILD_NUMBER} \
+          --build-arg commit=${GIT_COMMIT_HASH} \
+          --file Containerfile \
+          .
+
+      - name: Save Docker Image
+        run: docker save --output docker-image.tar ${{ env.CONTAINER_REGISTRY_URL }}/${{ env.APPLICATION_NAME }}
```

Comment:
Yes, it now pushes the whole thing with all the tags it was built with (what I understood from https://docs.docker.com/engine/reference/commandline/save/)

Reply:
Ah sorry, my bad. My comment was actually about the `docker save` command. You have changed that from `docker save --output docker-image.tar ${DOCKER_IMAGE_TAG}` to `docker save --output docker-image.tar ${{ env.CONTAINER_REGISTRY_URL }}/${{ env.APPLICATION_NAME }}` and I was wondering why? I wonder if this will not cause problems maybe?